lib/ansible/plugins/connection/ssh.py

```diff
@@ -48,6 +48,17 @@
               - name: ansible_password
               - name: ansible_ssh_pass
               - name: ansible_ssh_password
+      sshpass_prompt:
```

Comment:
version_added for the top entry, not just the var

Reply:
Oops, fixed.